### ISSUE TITLE
invite buttons for community owner is gone in the settings

### ISSUE THREAD

**Issue by iurimatias** [status-im/status-desktop#12101]
# Bug Report

## Description

![Screen Shot 2023-09-07 at 9 04 09 AM](https://github.com/status-im/status-desktop/assets/176720/d8650928-1c14-40bb-8915-a3e208862422)

## Steps to reproduce

1. go to a community you are the owner or control node
2. go to community management and overview

expected result: there are the buttons or some method to invite users to the community
current result: those buttons are gone, and there is no way in the settings to invite someone

**Comment by iurimatias** [comment]
Ok so it seems likely the idea is that the right click menu is the intended way to invite users, can you confirm if that's the case @benjthayer

**Comment by benjthayer** [comment]
> Ok so it seems likely the idea is that the right click menu is the intended way to invite users, can you confirm if that's the case @benjthayer

Correct. The community admin overview screen was cleaned up and the CTA to invite user removed given the user can invite people to the community in other ways including right-clicking the Community name, community icon, channel area and accessing the share link when selecting `View community` when right-clicking the Community icon in the left side App/Community bar. Personally, I think we would benefit from a menu accessed by the Community name much like Discord has where a user could access notification settings, invite people etc but for now, this is the design intent yes. 

https://www.figma.com/file/qHfFm7C9LwtXpfdbxssCK3/Kuba%E2%8E%9CDesktop---Communities?type=design&node-id=2771-384218&mode=design&t=8SsfD1Yqvy4e6hqQ-4
